### ISSUE TITLE
WinPython 2.7.10.2-3: "cx_Freeze" is broken

### ISSUE THREAD

**Issue by PierreRaybaut** [winpython/winpython#210]
Here is the traceback when trying to use cx_Freeze in WinPython 2.7.10.2 or 2.7.10.3 (not tested with 2.7.10.1 yet):
```python
  File "C:\Temp\WinPython-64bit-2.7.10.2\python-2.7.10.amd64\lib\site-packages\cx_Freeze\dist.py", line 362, in setup
    distutils.core.setup(**attrs)
  File "C:\Temp\WinPython-64bit-2.7.10.2\python-2.7.10.amd64\lib\distutils\core.py", line 151, in setup
    dist.run_commands()
  File "C:\Temp\WinPython-64bit-2.7.10.2\python-2.7.10.amd64\lib\distutils\dist.py", line 953, in run_commands
    self.run_command(cmd)
  File "C:\Temp\WinPython-64bit-2.7.10.2\python-2.7.10.amd64\lib\distutils\dist.py", line 972, in run_command
    cmd_obj.run()
  File "C:\Temp\WinPython-64bit-2.7.10.2\python-2.7.10.amd64\lib\distutils\command\build.py", line 127, in run
    self.run_command(cmd_name)
  File "C:\Temp\WinPython-64bit-2.7.10.2\python-2.7.10.amd64\lib\distutils\cmd.py", line 326, in run_command
    self.distribution.run_command(command)
  File "C:\Temp\WinPython-64bit-2.7.10.2\python-2.7.10.amd64\lib\distutils\dist.py", line 972, in run_command
    cmd_obj.run()
  File "C:\Temp\WinPython-64bit-2.7.10.2\python-2.7.10.amd64\lib\site-packages\cx_Freeze\dist.py", line 232, in run
    freezer.Freeze()
  File "C:\Temp\WinPython-64bit-2.7.10.2\python-2.7.10.amd64\lib\site-packages\cx_Freeze\freezer.py", line 626, in Freeze
    self.compress, self.copyDependentFiles)
  File "C:\Temp\WinPython-64bit-2.7.10.2\python-2.7.10.amd64\lib\site-packages\cx_Freeze\freezer.py", line 526, in _WriteModules
    module.Create(finder)
  File "C:\Temp\WinPython-64bit-2.7.10.2\python-2.7.10.amd64\lib\site-packages\cx_Freeze\freezer.py", line 762, in Create
    module.file, module.name)
cx_Freeze.freezer.ConfigError: no file named sys (for module collections.sys)
```

Initially, I thought that the package `backports-abc` introduced in WinPython 2.7.10.3 was the cause but it's not. Removing it does not solve this issue.

FYI, a similar bug was found with Python 3.4.0 (and `collections.abc`) but was resolved: this issue does not concern WinPython 3.4.3.6, for example (I've tested it).

**Comment by PierreRaybaut** [comment]
I found out where it comes from: removing `jsonschema` package fixes this issue (`jsonschema‑2.5.1‑py2.py3‑none‑any.whl`).

**Comment by stonebig** [comment]
WinPython 2.7, and all Python2.7 issues, are no more supported. Please switch to WinPython 3.4+ or to another Python Distribution.
